service_management/test/services/analysis/SampleObjectServiceSpec.scala

```diff
@@ -114,6 +115,10 @@ class SampleObjectServiceSpec
       val so    = service.delete(found.objectId.get).futureValue
       so mustBe MusitSuccess(())
     }
+    "not find a deleted sample" in {
```

Comment:
Can you add a newline between the tests here and remvoe the extra newline at line 88.